lib/src/components/progress/progress_circle.dart

```diff
@@ -0,0 +1,127 @@
+import 'dart:math' as math;
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+import '../../../zeta_flutter.dart';
+import 'progress.dart';
+
+/// Sizes for [ZetaProgressCircle]
+enum ZetaCircleSizes {
```

Comment:
Is there any way we can use a more generic enum for this like ZetaWidgetSize?

Reply:
Circle sizes have more options so i could use an extension on ZetaWidgetSize , but it feels unnecessary to only extend 3 sizes.